test/traverse/test.ml

```diff
@@ -82,3 +82,60 @@ class virtual iter :
     method t : t -> unit
   end
 |}]
+
+type t =
+  | X
+  | Arrow of { label : string option; domain : t; range : t }
```

Comment:
Very small nitpick: the test could be made more "to the point".
```suggestion
type t = Inline of { a : string option; b : t }
```

Reply:
Changed, thanks!